src/DropdownMenu.js

```diff
@@ -188,17 +177,13 @@ class DropdownMenu extends React.Component {
   }
 }
 
-const DecoratedDropdownMenu = mapContextToProps(
-  DropdownContext,
-  ({ show, alignEnd, toggle, drop, menuRef, toggleNode }, props) => ({
-    drop,
-    menuRef,
-    toggleNode,
-    onToggle: toggle,
-    show: show == null ? props.show : show,
-    alignEnd: alignEnd == null ? props.alignEnd : alignEnd,
-  }),
-  DropdownMenu,
-);
-
-export default DecoratedDropdownMenu;
+const f = ({ show, alignEnd, toggle, drop, menuRef, toggleNode }, props) => ({
```

Comment:
we should be exporting this `f`? or at least using it?

Reply:
or maybe we're not using this at all?

Reply:
just left over while refactoring oops